src/js/pensions/components/GetFormHelp.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function GetFormHelp() {
+  return (
+    <div>
+      <p className="talk">For benefit related questions please call:</p>
+      <p className="phone-number">
+        <a href="tel:+800-827-100">800-827-100</a><br/>
```

Comment:
It looks like we've been including the `+` here which is not required, but not the country code that would go along with it. Should we include the country code? or remove the `+`?